bower.json

```diff
@@ -22,6 +22,11 @@
     "test",
     "tests"
   ],
+  "dependencies": {
+    "angular": "~1.4.4",
+    "angular-sanitize": "~1.4.4",
```

Comment:
Ah, okay...I'm new to the pull-request thing, so I must have mis-read.  I appreciate you being open to this change, your library here is really handy.